addon/pods/components/rui-validatable-input/template.hbs

```diff
@@ -11,9 +11,8 @@ name=valuePath}}
   </p>
 {{/if}}
 
-{{#if showMessage}}
+{{#if isInvalid}}
```

Comment:
I am still working on the styling for this and will be included in a separate pr.